### ISSUE TITLE
gameRef.pauseEngine not working when calling gameRef.overlays.add('ID') afterwards.

### ISSUE THREAD

**Issue by fly-qp** [flame-engine/flame#1083]
Version: 1.0.0-releasecandidate.16

Calling these methods on pause button:

`
game.pauseEngine();
game.overlays.add(PauseMenu.id);
game.overlays.remove(QuickSettings.id);
`

Without the line `game.overlays.add(PauseMenu.id)` it works like a charm. 
It seems this line reactivates the game loop back on. 

I couldn't figure it out what exactly in the library caused this problem.

# Workaround
Problem solved dirty with calling `game.pauseEngine();` again in the initState methode of PauseMenu.

**Comment by spydon** [comment]
This was fixed in #1066
You can depend on main until we release rc17:
```yaml
dependencies:
  flame:
    git:
      url: https://github.com/flame-engine/flame.git
      ref: main
      path: packages/flame
```

**Comment by luanpotter** [comment]
`1.0.0-releasecandidate.18` (and rc17) have been released!
